src/test/java/org/minions/devfund/carlosorellana/SquarelotronTest.java

```diff
@@ -0,0 +1,162 @@
+        assertArrayEquals(expectedMatrix, squarelotron1.getSquarelotron());
+    }
+
+    /**
+     * Verify that UpsideDowFlip method return the expected result when select the Ring 2.
+     */
+    @Test
+    public void testUpsideDownFlipMethodRingTwo() {
+        final int[][] expectedMatrix = {
+                {1, 2, 3, 4, 5},
+                {6, 17, 18, 19, 10},
+                {11, 12, 13, 14, 15},
+                {16, 7, 8, 9, 20},
+                {21, 22, 23, 24, 25}
+        };
+
+        final int size = 5;
+        final int ring = 2;
+        Squarelotron squarelotron = new Squarelotron(size);
+        Squarelotron squarelotron1 = squarelotron.upsideDownFlip(ring);
+        System.out.println(squarelotron1.toString());
+        assertArrayEquals(expectedMatrix, squarelotron1.getSquarelotron());
+    }
+
+    /**
+     * Verify that MainDiagonalFlip method return the expected result when select the Ring 1.
+     */
+    @Test
+    public void testMainDiagonalFlipMethodRingOne() {
+        final int[][] expectedMatrix = {
+                {1, 6, 11, 16, 21},
+                {2, 7, 8, 9, 22},
+                {3, 12, 13, 14, 23},
+                {4, 17, 18, 19, 24},
+                {5, 10, 15, 20, 25}
+        };
+
+        final int size = 5;
+        final int ring = 1;
+        Squarelotron squarelotron = new Squarelotron(size);
+        Squarelotron squarelotron1 = squarelotron.mainDiagonalFlip(ring);
+        System.out.println(squarelotron1.toString());
+        assertArrayEquals(expectedMatrix, squarelotron1.getSquarelotron());
+    }
+
+    /**
+     * Verify that rotateRight method return the expected result when select the Ring 1.
+     */
+    @Test
+    public void testMainDiagonalFlipMethodRingTwo() {
+        final int[][] expectedMatrix = {
+                {1, 2, 3, 4, 5},
+                {6, 7, 12, 17, 10},
+                {11, 8, 13, 18, 15},
+                {16, 9, 14, 19, 20},
+                {21, 22, 23, 24, 25}
+        };
+        final int size = 5;
+        final int ring = 2;
+        Squarelotron squarelotron = new Squarelotron(size);
+        Squarelotron squarelotron1 = squarelotron.mainDiagonalFlip(ring);
+        System.out.println(squarelotron1.toString());
+        assertArrayEquals(expectedMatrix, squarelotron1.getSquarelotron());
+    }
+
+    /**
+     * Verify that MainDiagonalFlip method return the expected result when select the Ring 1.
+     */
+    @Test
+    public void testRotateRightMethodRotate90Grades() {
+        final int size = 5;
+        final int turn = 1;
+        final int[][] expectedMatrix = {
+                {21, 16, 11, 6, 1},
+                {22, 17, 12, 7, 2},
+                {23, 18, 13, 8, 3},
+                {24, 19, 14, 9, 4},
+                {25, 20, 15, 10, 5}
+        };
+
+        Squarelotron squarelotron = new Squarelotron(size);
+        squarelotron.rotateRight(turn);
+        System.out.println(squarelotron.toString());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)